spec/jobs/job_run_complete_job_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+RSpec.describe JobRunCompleteJob do
+  let(:job) { described_class.new }
+  let(:job_run) { create(:job_run, :preassembly, state:) }
+  let(:completed_accession) { create(:accession, :completed, job_run:) }
+  let(:failed_accession) { create(:accession, :failed, job_run:) }
+  let(:in_progress_accession) { create(:accession, :in_progress, job_run:) }
+
+  before do
+    job_run.accessions = accessions
+  end
+
+  context 'when all accessions are completed or failed for preassembly_complete' do
+    let(:accessions) { [completed_accession, failed_accession] }
+    let(:state) { 'preassembly_complete' }
+
+    it 'transitions the job run to completed' do
+      expect { job.perform(job_run) }.to change { job_run.reload.state }.from(state).to('accessioning_complete')
+    end
+  end
+
+  context 'when all accessions are completed or failed for preassembly_complete_with_errors' do
+    let(:accessions) { [completed_accession, failed_accession] }
+    let(:state) { 'preassembly_complete_with_errors' }
+
+    it 'transitions the job run to completed' do
+      expect { job.perform(job_run) }.to change { job_run.reload.state }.from(state).to('accessioning_complete')
```

Comment:
This is very readable! Is it possible/desirable to use the symbol instead of the 'accessioning_complete' string here?

Reply:
Didn't work:
```
Failure/Error: expect { job.perform(job_run) }.to change { job_run.reload.state }.from(state).to(:accessioning_complete)
       expected `job_run.reload.state` to have initially been :preassembly_complete, but was "preassembly_complete"
     # ./spec/jobs/job_run_complete_job_spec.rb:19:in `block (3 levels) in <top (required)>'
```

Symbols are one of the rubyisms that make not a lick of sense to me.